app/actions/CommonActions.scala

```diff
@@ -62,6 +62,29 @@ object CommonActions {
       case _ => _platform
     }
 
+    def playId: Option[String] = {
+      for {
+        playSessionCookie <- request.cookies.get("PLAY_SESSION")
+        value <- Option(playSessionCookie.value)
+      } yield {
+        value.take(value.indexOf("-"))
+      }
+    }
+
+    def initialSessionId: String = {playId.getOrElse("")}
+
+    def sessionId: String = {
+      val payment_session = request.session.get("payment_session")
+      payment_session match {
+        case Some(id) =>
```

Comment:
If `initialSessionId` returned a non-empty string for missing values, this case statement could be rewritten as: `paymentSession.getOrElse(initialSessionId)`

Reply:
Good idea! Fixed in next commit.